tests/integration/utils/fixture.mjs

```diff
@@ -29,7 +30,7 @@ export const setupFixtureTests = async function (options, factory) {
 
   beforeAll(async () => {
     if (options.fixture) fixture = await Fixture.create(options.fixture)
-    if (options.devServer) devServer = await startDevServer({ cwd: fixture.directory })
+    if (options.devServer) devServer = await startDevServer({ cwd: fixture.directory, args: ['--offline'] })
```

Comment:
when we run fixture tests we should never read settings from the API.

Reply:
good point!